.travis.yml

```diff
@@ -4,6 +4,15 @@ python:
   - "2.7"
   - "3.4"
   - "3.5"
+  - "3.6"
+  - "nightly"  # points to 3.7 right now (2016-12-28)
```

Comment:
Okay...  So I'm happy to remove it, but I'm not sure if you noticed the second grouping of code:

```
matrix:
  fast_finish: true
  allow_failures:
    - python: "3.6"
    - python: "nightly"
```

By adding the extra matrix of tests, you can see when things start to pass (or do not pass) with your setup.  You can see that output here:  https://travis-ci.org/dgilland/alchy/builds/187805406

More or less, python 3.6 and 3.7 (dev branch) fail, but your travis build still passes. The matrix setup allows python 3.6 and nightly to fail.  This may add maybe a couple of extra seconds to the build time on travis, but for the most part, it's a negligible performance impact.  However, adding the tests adds nice development level of detail that you cannot immediately find without setting up locally and running the tests and then attempting to fix.